data/articles/html/360004346254-Workflow-status-checks-never-completes-because-of-cicircleci--Waiting-for-status-to-be-reported_en-us.html

```diff
@@ -1,3 +1,2 @@
 <p>When adding branch protections to your workflow jobs you'll need to make sure the ci/circleci check is unchecked, as it will never report back with a status. Review your branch protection settings and make sure it is deselected.</p>
 <p><img src="https://circleci.com/docs/assets/img/docs/github_branches_status.png" alt="Uncheck GitHub Status Keys"></p>
-<p><br>Read in more detail below about why this can cause issues<br><a href="https://circleci.com/docs/2.0/workflows-waiting-status/" target="_blank" rel="noopener">https://circleci.com/docs/2.0/workflows-waiting-status/</a></p>
```

Comment:
<p><br>Read in more detail below about why this can cause issues<br><a href="https://circleci.com/docs/2.0/workflows/#workflows-waiting-for-status-in-github/" target="_blank" rel="noopener">https://circleci.com/docs/2.0/workflows/#workflows-waiting-for-status-in-github/</a></p>